README.md

```diff
@@ -21,23 +21,23 @@ Device ID 1: NZXT Kraken X (X42, X52, X62 or X72)
 
 # liquidctl status
 NZXT Smart Device (V1)
-├── Fan 1                        PWM  
```

Comment:
This matched the real output from `status`, which ends up with these trailing spaces where the units is blank.